.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
Aye, got it. Sorry. Those `METADATA_PROCESSORS` have to be made more robust, for sure.